assets/src/stories-editor/helpers/index.js

```diff
@@ -479,24 +479,31 @@ export const wrapBlocksInGridLayer = ( element, blockType, attributes ) => {
 	const style = {
 		style: {},
 	};
+	const isTextBlock = ( 'amp/amp-story-text' === blockType.name );
```

Comment:
@kienstra The border changes should be for the editor only, not sure why do we need to change anything for saving into the post content. Thoughts? The compensating would ideally influence the editor only since the dragging also influences the editor only.

Reply:
By influencing the editor only I'm thinking that maybe it would make more sense to adjust the positioning in the editor only, for example, to compensate in the dragging logic instead? Otherwise, if for some reason we would want to change the draggable area's size then it would break all the blocks.

Also, we should probably also compensate for the lost weight and height in the editor.

Without border:
![Screen Shot 2019-06-04 at 4 37 18 PM](https://user-images.githubusercontent.com/3294597/58888058-0de21b00-86e7-11e9-8ad6-dcfe944ee3a8.png)

With border:
![Screen Shot 2019-06-04 at 4 37 11 PM](https://user-images.githubusercontent.com/3294597/58888050-0ae72a80-86e7-11e9-9924-93fc2dc0a191.png)

As you can see the border has taken away some of the area that the block previously had. Perhaps we should add this back by displaying the block a bit larger, just in the editor?